students/EricAdams/trapezoidal_rule/decorator_trapezoidal_rule.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python
+
+import trapezoidal_rule as tr
+
+
+def area_under_function(func):
```

Comment:
I hadn't thought to make a decorator out of this -- interesting!

but you don't need to re-implement -- you could call `trapz_area_under_curve` from in the decorator.